docs/readthedocs/source/doc/PPML/QuickStart/tpc-ds_with_sparksql_on_k8s.md

```diff
@@ -0,0 +1,231 @@
+## TPC-DS with Trusted SparkSQL on Kubernetes
+
+### Prerequisites
+
+- Hardware that supports SGX
+- A fully configured Kubernetes cluster
+- Intel SGX Device Plugin to use SGX in K8S cluster (install following instructions [here](https://bigdl.readthedocs.io/en/latest/doc/PPML/QuickStart/deploy_intel_sgx_device_plugin_for_kubernetes.html "here"))
+
+### Prepare TPC-DS kit and data
+
+1. Download and compile tpc-ds
+
+```
```

Comment:
git clone zoo-tutorial, then cd cd /path/to/zoo-tutorials
Then, git clone https://github.com/databricks/tpcds-kit.git and make